x/community/keeper/keeper_test.go

```diff
@@ -89,3 +92,56 @@ func (suite *KeeperTestSuite) TestGetAndSetStakingRewardsState() {
 
 	suite.Equal(validParams, keeper.GetStakingRewardsState(suite.Ctx), "expected fetched state to equal set state")
 }
+
+func (suite *KeeperTestSuite) TestGetAuthority_Default() {
+	suite.Equal(
+		authtypes.NewModuleAddress(govtypes.ModuleName),
+		suite.Keeper.GetAuthority(),
+		"expected fetched authority to equal x/gov address",
+	)
+}
+
+func (suite *KeeperTestSuite) TestGetAuthority_Any() {
+	tests := []struct {
+		name      string
+		authority sdk.AccAddress
+	}{
+		{
+			name:      "gov",
+			authority: authtypes.NewModuleAddress(govtypes.ModuleName),
+		},
+		{
+			name:      "empty",
+			authority: sdk.AccAddress{},
+		},
```

Comment:
Do we want to allow empty address for authority?

Reply:
Could validate the address format in `NewKeeper` with `sdk.VerifyAddressFormat`.